cores/esp8266/time.cpp

```diff
@@ -74,6 +75,23 @@ void configTime(int timezone_sec, int daylightOffset_sec, const char* server1, c
     sntp_init();
 }
 
+extern "C++"
+void configTime(const char* tz, const char* server1, const char* server2, const char* server3)
+{
+    sntp_stop();
+
+    setServer(0, server1);
```

Comment:
Should an assert be added here to check against all three servers being nullptr or empty strings?

Reply:
Does setServer() return a value? If so, it could return an error status.
My preference is usually to return a exception status rather than crash the system.

Reply:
`setServer()` checks for nullptr, and also lwIP does.
So there is a double check, but I'll leave the one in setServer anyway.